src/consts/interfaces.consts.ts

```diff
@@ -44,8 +44,11 @@ export interface ICreatedPeanutLink {
 	txHash: string
 }
 
-export interface IPeanutUnsignedTransactions {
-	unsignedTxs: TransactionRequest // change this any type to correct type
+export interface IPeanutUnsignedTransaction {
+	to: string
+	nonce: number
+	data: string
+	value: BigInt
```

Comment:
just a comment that transactions in the ethereum sense also have other various other fields (gas params, storage access params, chainid etcetc)

could potentially be slightly confusing, but I think this is a minor gripe. Good as is imo

Reply:
Yes, there are a lot of other fields but we do not populate these fields, the ones that are defined here are the only ones that are populated.